src/store/modules/invites.js

```diff
@@ -0,0 +1,61 @@
+import Vue from 'vue';
+import { Crypto } from '@aeternity/aepp-sdk/es';
+import MemoryAccount from '@aeternity/aepp-sdk/es/account/memory';
+import { AE_AMOUNT_FORMATS } from '@aeternity/aepp-sdk/es/utils/amount-formatter';
+
+export default {
+  namespaced: true,
+  state: {
+    referrals: [],
+  },
+  getters: {
+    get: state => link => state.referrals.find(r => r.link === link),
+    keypair: () => referral => {
```

Comment:
Is this getter used somewhere?

Reply:
In the beginning it was used, but currently is not. I decided to leave it because it can be used for extracting keypair from url 🤷‍♀️ 
> Opening such link (deeplink)
> open a new view which will decode the url - and claim the balance to your account

Reply:
After removing we will be able to get it back (when needed) from git history✂️